internal/static/internal/css/frontpage.css

```diff
@@ -119,3 +119,61 @@
     margin-bottom: 0.4em;
     box-sizing: border-box;
 }
+
+/* kode for slideshow */
+@keyframes slidy {
```

Comment:
comment could be replaced with self-documenting naming, meanig that the slidy -> slideshow or something like that making it a bit more obvious (although slidy is a catchy name).